_posts/2022-06-30-blog-overview-of-wpe.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: "An overview of the WPE WebKit project"
+tags: [blogpost]
+author: csaavedra
+permalink: /blog/02-overview-of-wpe.html
+---
+
+In <a href="https://wpewebkit.org/blog/01-happy-birthday-wpe.html">the
```

Comment:
Could we make this link only have an absolute path? I just realized that's a good practice that plays better with the site preview feature we have for pull requests 🔨 

```suggestion
In <a href="/blog/01-happy-birthday-wpe.html">the
```